api/src/main/java/org/openmrs/api/impl/PatientServiceImpl.java

```diff
@@ -1319,7 +1319,7 @@ public IdentifierValidator getIdentifierValidator(String pivClassName) {
 		}
 		catch (ClassNotFoundException e) {
 			log.error("Could not find patient identifier validator " + pivClassName, e);
```

Comment:
Should we really log this exception if we are throwing it?

Reply:
I remove the log this exception.